submarine-server/server-core/src/test/java/org/apache/submarine/server/rest/MetaStoreApiTest.java

```diff
@@ -42,7 +42,7 @@
 import static org.junit.Assert.assertTrue;
 
 public class MetaStoreApiTest {
```

Comment:
It's better to rename MetaStoreApiTest to MetaStoreRestApiTest

Reply:
Done.